src/index.html

```diff
@@ -8,6 +8,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars-block">
+      <div class="stars stars--0">
+        <ul class="stars-item">
```

Comment:
no need in this additional class
and you can use only <div> instead of unnecessary ul and li